Frontend/src/classes/AdminManager.js

```diff
@@ -0,0 +1,9 @@
+import BasicUserManager from './BasicUserManager'
+class AdminManager extends BasicUserManager{
+    constructor(username, users){
+        super(username);
+        this.users = users;
```

Comment:
Why there are multiple users in admin? or what is this users variable?

Reply:
Initially I planned to have a list of all the users an admin should be able to view, but we ended up agreeing to just let the admin access all the end users. I just forgot to remove the list from the admin. I'll remove it.